packages/driver/src/cypress/stack_utils.ts

```diff
@@ -146,6 +199,11 @@ const getInvocationDetails = (specWindow, sourceMapProjectRoot: string): Invocat
       }
     }
 
+    // if the hook is the test, and it's an e2e test,, we will try to remove the lines that are not the actual invocation of the test
+    if (type === 'test' && specWindow.Cypress.testingType !== 'component') {
```

Comment:
```suggestion
    if (type === 'test' && specWindow.Cypress.testingType === 'e2e') {
```
Also move this `if` statement into the above `if (specWindow.Cypress)` statement.

Reply:
updated 👍🏻